homeassistant/components/yolink/sensor.py

```diff
@@ -161,7 +191,11 @@ def __init__(
     @callback
     def update_entity_state(self, state: dict) -> None:
         """Update HA Entity State."""
-        self._attr_native_value = self.entity_description.value(
-            state.get(self.entity_description.key)
-        )
+        if (
+            attr_val := self.entity_description.value(
+                state.get(self.entity_description.key)
+            )
+        ) is None and self.entity_description.key in ["devTemperature", "loraInfo"]:
```

Comment:
It's better to add an optional attribute to the entity description that holds a function that we can call to see if we should return early here.

Reply:
thanks for advise, I will open a new PR for this.